tasks/main.yml

```diff
@@ -58,6 +58,20 @@
   loop_control:
     label: "{{ item.section }}:{{ item.option }}"
 
+- name: Place TLS certificates
+  ansible.builtin.copy:
+    src: "{{ item.src }}"
+    dest: "{{ item.dst }}"
+    owner: bareos
+    group: bareos
+    mode: "0640"
+    backup: "{{ bareos_dir_backup_configurations }}"
+  loop:
+    - { src: "ca.pem", dst: "{{ bareos_webui_tls_ca_cert }}" }
```

Comment:
Where are theses source files coming from? We shouldn't assume that they are just there.
I'd rather go with an additional var in `defaults/main.yml` to specify the source file name. you could update the now existing `ca_cert`/`cert_chain` to something like this:
```yaml
bareos_webui_tls_ca_cert:
  src: ""
  dest: ""
```

Reply:
Here we could also define a "biased" destination path like `"/etc/bareos/bareosCA.pem"` so the end-user just has to care about having the files on the Ansible controller and can change the destination only if he wants to.